### PR TITLE
Vet clinic database: query and update animals table

### PR DESCRIPTION
**Pull request by kalbek**
## Changes made in this PR:
- add ```species``` column to ```animals``` table
- insert data to ``` data.sql``` file
- update the ```animals``` table using transactions
- write queries for specific questions
- Take screenshots of transaction actions
#### -- Screenshot 1 --
Sc-1-Shell (psql)
#### -- Screenshot 2 --
Sc-2-Shell (psql)
#### -- Screenshot 3 --
Sc-3-Shell (psql)
#### -- Screenshot 4 --
Sc-4-Shell (psql)
#### -- Screenshot 5 (Confirm all records are deleted.) --
Sc-5-Shell (psql)

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,10 +1,68 @@
...
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+-- Update the animals table by setting the species column to 'pokemon' for all animals that don't have species already set.
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+-- Verify that changes were made.
+SELECT * FROM animals;
+-- Commit the transaction.
+COMMIT;
+-- Verify that changes persist after commit.
+SELECT * FROM animals;
```
Comment (Kamba56) [inline queries.sql:12]: - [ ] Before setting the species of the animals, kindly Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction. Also add screenshots of results you got from this to the pull request description as demanded by project requirements.

File: queries.sql [hunk 2]
```diff
@@ -1,10 +1,68 @@
...
+
+-- Start a transaction
+BEGIN;
+
+-- Delete all records in the animals table.
+DELETE FROM animals;
+
+-- Roll back the transaction.
```
Comment (Kamba56) [inline queries.sql:20]: - [ ] Kindly add a confirmation that all animals were deleted by selecting all before rollback and add a screenshot.

File: queries.sql [hunk 3]
```diff
@@ -1,10 +1,68 @@
...
+SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
+
+-- Avg weight of animals
+SELECT AVG(weight_kg) FROM animals;
+
+-- Who escaped the most
+SELECT neutered, COUNT(*) FROM animals WHERE escape_attempts > 0 GROUP BY neutered;
+
+-- Min and Max weight of each type of animals
+SELECT name, MIN(weight_kg), MAX(weight_kg) FROM animals GROUP BY name;
```
Comment (Kamba56) [inline queries.sql:61]: - [ ] This query should select the minimum and maximum weight of each type of animal. remember there are two types of animals, Digimon and Pokemon, those are the types being referred to here. Kindly update your query.

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @kalbek, 

Good job so far!!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [x] Great job adding the species to all animals!👏👏👏
- [x]

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- Your project meets the requested changes by the previous reviewer :heavy_check_mark: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**